packages/grafana-ui/src/components/Drawer/Drawer.tsx

```diff
@@ -264,10 +274,22 @@ const getStyles = (theme: GrafanaTheme2) => {
       position: 'relative',
     }),
     drawer: css({
+      // Override rc-drawer's default position: fixed to position: absolute
+      // so it positions relative to its container (.page-panes) instead of viewport
+      // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+      position: 'absolute !important' as 'absolute',
+      inset: 0,
       top: 0,
+      bottom: 0,
+      left: 0,
+      right: 0,
 
       '.rc-drawer-content-wrapper': {
         boxShadow: theme.shadows.z3,
+        position: 'absolute',
+        right: 0,
+        top: 0,
+        bottom: 0,
```

Comment:
when it was like this before (below top nav), it was like this, https://github.com/grafana/grafana/pull/93725/files 

seems like a smaller change. (and it should be even smaller as it only extends below the first level nav not the second (if it exists), like before which added some complexity